### PR TITLE
[feature/breaking-changes]: Cleanup build

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.FluentTheme.v1/FluentMerge.targets [hunk 1]
```diff
@@ -35,7 +35,7 @@
 	<Message Text="Theme resources XAML file was generated" />
   </Target>
 
-  <PropertyGroup Condition="'$(TargetFramework)'=='netstandard2.0' and '$(UnoRuntimeIdentifier)'=='Reference'">
```
Comment (Youssef1313) [inline src/Uno.UI.FluentTheme.v1/FluentMerge.targets:38]: This is a potential behavior change. `$(TargetFramework)` is supposed to never be `netstandard2.0`.

File: src/Uno.UI.RemoteControl.Host/RemoteControlServer.cs [hunk 2]
```diff
@@ -131,12 +131,10 @@ private void ProcessDiscoveryFrame(Frame frame)
 				// As BasePath is a directory, try and load processors from assemblies within that dir
 				var basePath = msg.BasePath.Replace('/', Path.DirectorySeparatorChar);
 
-#if NET7_0_OR_GREATER
+#if NET8_0_OR_GREATER
```
Comment (Youssef1313) [inline src/Uno.UI.RemoteControl.Host/RemoteControlServer.cs:134]: @jeromelaban Is this change correct?

File: src/Uno.UI.RemoteControl/Uno.UI.RemoteControl.Wasm.csproj [hunk 3]
```diff
@@ -17,7 +17,7 @@
 
 		<Nullable>enable</Nullable>
 
-		<UnoRuntimeIdentifier Condition="'$(TargetFramework)'=='netstandard2.0'">WebAssembly</UnoRuntimeIdentifier>
+		<UnoRuntimeIdentifier>WebAssembly</UnoRuntimeIdentifier>
```
Comment (Youssef1313) [inline src/Uno.UI.RemoteControl/Uno.UI.RemoteControl.Wasm.csproj:20]: This is a behavioral change again. Previously `UnoRuntimeIdentifier` is not set for this project.

File: src/Uno.UI/Extensions/UIElementExtensions.cs [hunk 4]
```diff
@@ -104,9 +104,6 @@ static string GetNumberedColumnIndentation(int depth)
 			elt switch
 			{
 				null => 0,
-#if NETSTANDARD
```
Comment (jeromelaban) [inline src/Uno.UI/Extensions/UIElementExtensions.cs:107]: I wonder if it should always be enabled. It's not super important though as it's a debug helper.

File: src/Uno.UI/UI/Xaml/DragDrop/DragDropExtension.wasm.cs [hunk 5]
```diff
@@ -30,9 +30,7 @@
 using Uno.UI.Xaml;
 using System.Diagnostics.CodeAnalysis;
 
-#if NET7_0_OR_GREATER
```
Comment (jeromelaban) [inline src/Uno.UI/UI/Xaml/DragDrop/DragDropExtension.wasm.cs:33]: We'll probably need to make another pass at this once @ebariche is done with CSP.

File: src/Uno.UI/Extensions/UIElementExtensions.cs (hunk 4 above)
Comment (Youssef1313) [reply src/Uno.UI/Extensions/UIElementExtensions.cs:107]: Added back with corrected condition.

**Review by jeromelaban — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/DragDrop/DragDropExtension.wasm.cs (hunk 5 above)
Comment (Youssef1313) [reply src/Uno.UI/UI/Xaml/DragDrop/DragDropExtension.wasm.cs:33]: Sure. @ebariche Let me know once the work is done and I'll do it.

**Pull request merged**